chains/substrate/writer/writer.go

```diff
@@ -51,7 +50,7 @@ func (w *SubstrateWriter) RegisterHandler(t relayer.TransferType, handler Propos
 func (w *SubstrateWriter) VoteProposal(m *relayer.Message) error {
 	handler, ok := w.handlers[m.Type]
 	if !ok {
-		return errors.New(fmt.Sprintf("no corresponding substrate handler found for message type %s", m.Type))
+		return fmt.Errorf("no corresponding substrate handler found for message type %s", m.Type)
```

Comment:
If the above `VoteProposal` method name is changed, then this should should also be changed for consistency

Reply:
did not get what you mean

Reply:
If `func (w *EVMVoter) VoteProposal(m *relayer.Message)` method name is changed then this method name should also be changed for consistency

`func (w *SubstrateWriter) VoteProposal(m *relayer.Message)`

Both method names could be modified so that it is clearer what they are doing (ie, `VoteOnProposal`)

Reply:
But method name did not changed, it i still VoteProposal in both cases

Reply:
Yeah, I was just suggesting we probably should change it